az-digital/az_quickstart
# Closes #1118 - Add Data Studio dashboard link to admin toolbar.

## Description
This PR adds a link to the admin toolbar to take authenticated users to the Universal Analytics Dashboard created by MBM. It should take editors/admins to an pre-filtered version (thanks to @trackleft!) of the dashboard based on their site url (assuming they are using the comprehensive analytics account)

## Related issues
Closes #1118 

## How to test
Log into the Probo site
Click on the Analytics Dashboard link
Verify that the dashboard is prefiltered based on the site url

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
